datalake/src/main/java/com/sequenceiq/datalake/job/SdxClusterStatusCheckerJob.java

```diff
@@ -74,6 +74,11 @@ protected void executeTracedJob(JobExecutionContext context) {
 
     private void syncSdxStatus(JobExecutionContext context) {
         getCluster().ifPresent(sdx -> {
+            if (!getRemoteResourceCrn().equals(sdx.getStackCrn())) {
+                // During processes such as resize the DL's stack CRN is modified, this ensures that it's updated in this job.
```

Comment:
I'm not an expert in this job, but shouldn't this job be unscheduled, then scheduled with the new crn instead of this?
cc @topolyai5 @bbihari @foldik

Reply:
Yes I can do that, I think I was originally wanting this to always be up to date to make it a more correct approach, but I definitely see that there could be some unintended consequences with this, as well as limiting how we manage the SDX stuff.
I will schedule a new job which is incredibly simple and overrides any pre-existing job with the same DL ID.